packages/react-dom/src/__tests__/ReactCompositeComponent-test.js

```diff
@@ -475,7 +476,8 @@ describe('ReactCompositeComponent', () => {
     expect(() => {
       instance = ReactDOM.render(<Component />, container);
     }).toWarnDev(
-      'Warning: setState(...): Cannot call setState() inside getChildContext()',
+      'Warning: setState(...): Cannot call setState() inside getChildContext().\n' +
+        '    in Component (at **)',
```

Comment:
Let’s rename it in the test so it’s more obvious we have it in the stack. Like MyComponent